docs/multicluster/user-guide.md

```diff
@@ -424,72 +438,64 @@ spec:
 created correctly, you can follow the [Multi-cluster Service](#multi-cluster-service)
-section to create Multi-cluster Services and verify cross-cluster Service
+section to create multi-cluster Services and verify cross-cluster Service
 access.
 
 ## Multi-cluster Service
 
-After you set up a ClusterSet properly, you can simply create a `ServiceExport` resource
-as below to export a `Service` from one member cluster to other members in the ClusterSet,
-you can update the name and Namespace according to your local K8s Service.
+After you set up a ClusterSet properly, you can create a `ServiceExport` CR to
+export a Service from one cluster to other clusters in the Clusterset, like the
+example below:
 
 ```yaml
 apiVersion: multicluster.x-k8s.io/v1alpha1
 kind: ServiceExport
 metadata:
   name: nginx
-  namespace: kube-system
+  namespace: default
 ```
 
-For example, once you export the `kube-system/nginx` Service in the member cluster
-`test-cluster-west`, Antrea Multi-cluster Controller in the member cluster will create
-two corresponding `ResourceExport` resources in the leader cluster, and the controller
-in leader cluster will create two `ResourceImport` contains all exported Service and
-Endpoints'. you can check the created resources in the leader cluster which should be
-like below:
+For example, once you export the `default/nginx` Service in member cluster
+`test-cluster-west`, it will be exported to member cluster `test-cluster-east`
+through the leader cluster. A Service and an Endpoints with name
+`default/antrea-mc-nginx` will be created in `test-cluster-east`, as well as
+a ServcieImport CR with name `default/nginx`. The Service `antrea-mc-nginx`
+contains the Endpoints exported from `test-cluster-west`, and any request to the
+Service from `test-cluster-east` will be routed to the backend `nginx` Pods in
+`test-cluster-west` through the Multi-cluster Gateways.
+
+As part of the Service export/import process, in the leader cluster, two
+ResourceExport CRs will be created in the Multi-cluster Controller Namespace,
+for the exported Service and Endpoints respectively, as well as two
+`ResourceImport` CRs. You can check them in the leader cluster with commands:
 
 ```sh
-$kubectl get resourceexport
+$kubectl get resourceexport -n antrea-multicluster
 NAME                                        AGE
 test-cluster-west-default-nginx-endpoints   7s
 test-cluster-west-default-nginx-service     7s
 
-$kubectl get resourceimport
+$kubectl get resourceimport -n antrea-multicluster
 NAME                      AGE
 default-nginx-endpoints   99s
 default-nginx-service     99s
 ```
 
-Then you can go to the member cluster `test-cluster-east` to check the new created
-Service and Endpoints with name `kube-system/antrea-mc-nginx` and a ServiceImport named
-`kube-system/nginx`. If there is already an existing Service created by users in the
-member cluster `test-cluster-east` also named `nginx` in Namespace `kube-system`, which
```

Comment:
Do you mean a Service with name `antrea-mc-nginx`? It is rather uncommon. Guess no need to mention that.